src/components/PivotTable/PivotTableRenderer/PivotTableRenderer.stories.tsx

```diff
@@ -0,0 +1,91 @@
+import React from 'react'
+import { KeyConfig } from '../model/model-keyMap'
+import { PivotTableProvider } from '../PivotTableCell/PivotTableProvider'
+import { PivotTableRenderer } from './PivotTableRenderer'
+import { TableProps } from './model-pivotTableRenderer'
+
+export default {
+  title: 'Components/PivotTable/PivotTableRenderer',
+}
+
+type Fruit = {
+  name: string
+  size: string
+}
+
+export const AllTypes = () => {
+  const props: TableProps<Fruit> = {
+    keysMapping: new Map([
```

Comment:
```suggestion
    keysMapping: new Map<keyof Fruit, KeyConfig>([
```
Then you don't need the explicit casting